.circleci/builder/factory.py

```diff
@@ -31,7 +31,7 @@ def obtain_options() -> argparse.Namespace:
     # Build Notebooks
     parser.add_argument('-c', '--notebook-collection-paths', type=str, default='')
     parser.add_argument('-n', '--notebook-category', type=str, default=None)
-    parser.add_argument('-r', '--remote-names', type=str, default='master')
+    parser.add_argument('-r', '--remote-names', type=str, default='main,master')
```

Comment:
Do we need to keep the old branch name?
```suggestion
    parser.add_argument('-r', '--remote-names', type=str, default='main')
```

Reply:
I'm honestly not sure. I'm not clear on exactly what this is used for... it seemed safer to do it this way? :shrug:

Reply:
Is @jbcurtin working on the machinery of CircleCI here; perhaps he can comment?

Reply:
Seems fine to me. 👍